database/users.ts

```diff
@@ -41,11 +42,12 @@ export async function getUserById(id: number) {
       id = ${id}
   `;
   return user;
-}
+});
 
-export async function getUserByValidSessionToken(token: string | undefined) {
-  if (!token) return undefined;
-  const [user] = await sql<User[]>`
+export const getUserByValidSessionToken = cache(
+  async (token: string | undefined) => {
+    if (!token) return undefined;
+    const [user] = await sql<User[]>`
     SELECT
```

Comment:
the SQL here is not indented properly anymore (may apply to multiple places in this PR)

Reply:
[a523b03](https://github.com/upleveled/security-vulnerability-examples-next-js-postgres/pull/85/commits/a523b03715231de100ca83b3456ece3d76983f1c)